BUILD.gn

```diff
@@ -519,15 +519,14 @@ if (is_mac) {
     extra_configs = [ "//build/config/compiler:wexit_time_destructors" ]
 
     deps = [
-      "//brave/vendor/sparkle:sparkle_resources_bundle_data",
       "//chrome:chrome_app_strings_bundle_data",
       "//chrome:chrome_resources",
       "//chrome:chrome_versioned_bundle_data",
       "//chrome/common:version_header",
     ]
 
-    if (is_chrome_branded) {
```

Comment:
this was removed because it serves no purpose here. `is_chrome_branded` will never be true for brave